service/controller/resource/configmap/desired.go

```diff
@@ -45,13 +48,37 @@ func (r *Resource) newConfigMaps(customResource v1alpha1.KVMConfig) ([]*apiv1.Co
 		return nil, microerror.Mask(err)
 	}
 
+	var release *releasev1alpha1.Release
+	{
+		releaseVersion := customResource.Labels[label.ReleaseVersion]
+		releaseName := fmt.Sprintf("v%s", releaseVersion)
+		release, err = r.g8sClient.ReleaseV1alpha1().Releases().Get(releaseName, apismetav1.GetOptions{})
```

Comment:
RBAC needs to be changed to allow us getting `Releases`

Reply:
Added